src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -665,11 +679,14 @@ void GeneralPage::initLayout(GeneralPageView &layout)
 
     layout.addCheckbox(
         "Only search for emote autocompletion at the start of emote names",
-        s.prefixOnlyEmoteCompletion);
+        s.prefixOnlyEmoteCompletion, false,
+        "When disabled, autocomplete emotes based on any part of the name.");
```

Comment:
I find this tooltip a tad confusing. I'm not sure how it should look like, but it took me a good 30s to understand what is being said here. @Felanbird could you help?

Reply:
```suggestion
        "When disabled, emote tab-completion will complete based on any part "
        "of the name."
        "\ne.g. sheffy -> DatSheffy");
```
minor wording change + an example, formatter doesn't mind what line the example is on 🤷